cmd/config-host-add.go

```diff
@@ -100,26 +117,32 @@ func checkConfigHostAddSyntax(ctx *cli.Context) {
 		fatalIf(errInvalidArgument().Trace(api),
 			"Unrecognized API signature. Valid options are `[S3v4, S3v2]`.")
 	}
+
+	if bucketlookup != "" && !isValidLookup(bucketlookup) {
```

Comment:
bucketlookup has already a default value (auto) and it will be better if we always check it here, so I guess we can remove `bucketlookup != ""` condition here